packages/synapse-react-client/src/components/Markdown/TOC.vitest.tsx

```diff
@@ -0,0 +1,81 @@
+import { MOCK_USER_ID } from '@/mocks/user/mock_user_profile'
+import { getUseQuerySuccessMock } from '@/testutils/ReactQueryMockUtils'
+import { createWrapper } from '@/testutils/TestingLibraryUtils'
+import { render, screen } from '@testing-library/react'
+import MarkdownSynapse from './MarkdownSynapse'
+import { vi, describe, beforeAll, it } from 'vitest'
+import {
+  useGetWikiPage,
+  useGetWikiAttachments,
+} from '@/synapse-queries/wiki/useWiki'
+
+vi.mock('@/synapse-queries/wiki/useWiki', () => ({
+  useGetWikiPage: vi.fn(),
+  useGetWikiAttachments: vi.fn(),
+}))
+
+const mockUseGetWikiPage = vi.mocked(useGetWikiPage)
+const mockUseGetWikiAttachments = vi.mocked(useGetWikiAttachments)
+
+describe('renders without crashing', () => {
```

Comment:
Updated to use Vitest and now mock data hooks instead of SynapseClient methods, but no other changes to these tests.